src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationQueryObjectType.cs

```diff
@@ -13,6 +18,27 @@ public LocalizationQueryObjectType(IStringLocalizer<LocalizationQueryObjectType>
 
             Field(x => x.Culture).Description(S["The culture for your content item."]);
             Field(x => x.LocalizationSet).Description(S["The localization set for your content item."]);
+
+            Field<ListGraphType<ContentItemType>, IEnumerable<ContentItem>>()
+                .Name("Localizations")
+                .Description(S["The localizations of the content item."])
+                .Argument<StringGraphType, string>("culture", "the culture of the content item", null)
+                .ResolveAsync( async x =>
+                {
+                    var culture = x.GetArgument<string>("culture");
+                    var context = (GraphQLContext)x.UserContext;
+                    var contentLocalizationManager = context.ServiceProvider.GetService<IContentLocalizationManager>();
+
+                    if (culture != null)
+                    {
+                        return new List<ContentItem>
+                        {
+                            await contentLocalizationManager.GetContentItemAsync(x.Source.LocalizationSet, culture)
```

Comment:
can this return null ? if so we're going to end up with a list with a null contentitem in it, as the field is not marked as nullable if this is the case we need to return an empty enumeration of contentitem

Reply:
I think it could. I now return an empty list instead